projects/graph/src/graph_demo.py

```diff
@@ -5,12 +5,70 @@
 """
 
 from sys import argv
+from graph import Graph
+from draw import BokehGraph
+import random
 
+def createDefaultGraph():
+    graph = Graph()  # Instantiate your graph
+    graph.add_vertex('0')
+    graph.add_vertex('1')
+    graph.add_vertex('2')
+    graph.add_vertex('3')
+    graph.add_edge('0', '1')
+    graph.add_edge('0', '3')    
 
-def main():
-    pass  # TODO
+    bg = BokehGraph(graph)
+    bg.draw()
+
```

Comment:
I think because you have lines 12-22 in here, the graph always has the same number of vertices and edges no matter what you enter as arguments when you run this in python. The vertices and edges appear in random locations on the screen, but I cannot generate a graph with, say, 8 vertices and 2 edges or something. This is just something to keep in mind - I think you have the idea on creating a random graph and should not spend too much time fixing this because DFS and BFS are super important and will appear on your sprint challenge.